### PR TITLE
Basics for rendering scenes as an embedded page

### PR DESCRIPTION
**Pull request by dprokop**

### CODE REVIEW

File: public/app/features/scenes/scenes/index.tsx [hunk 1]
```diff
@@ -10,32 +10,41 @@ import { getNestedScene } from './nested';
...
   ];
 }
 
 const cache: Record<string, Scene> = {};
 
-export function getSceneByTitle(title: string) {
+export function getSceneByTitle(title: string, standalone = true) {
   if (cache[title]) {
+    if (cache[title].state.standalone !== standalone) {
+      cache[title].setState({ standalone });
```
Comment (dprokop) [inline public/app/features/scenes/scenes/index.tsx:37]: This happens in ScenePage render, think for dev purposes it's fine, but in general,  setting Scene's state during render is an anti-pattern.

File: public/app/features/scenes/components/Scene.tsx [hunk 2]
```diff
@@ -35,7 +36,15 @@ export class Scene extends SceneObjectBase<SceneState> {
...
 
 function SceneRenderer({ model }: SceneComponentProps<Scene>) {
-  const { title, layout, actions = [], isEditing, $editor, subMenu } = model.useState();
+  const { title, layout, actions = [], isEditing, $editor, subMenu, standalone } = model.useState();
+
+  if (!standalone) {
+    return (
+      <div style={{ flexGrow: 1, display: 'flex', gap: '8px', overflow: 'auto', minHeight: '100%' }}>
+        <layout.Component model={layout} isEditing={isEditing} />
+      </div>
```
Comment (torkelo) [inline public/app/features/scenes/components/Scene.tsx:45]: this is missing the subMenu and toolbar actions, question is what to do with those. toolbar actions might not be needed. 

subMenu (variables, time picker) could be useful.

File: public/app/features/scenes/scenes/index.tsx (hunk 1 above)
Comment (torkelo) [reply public/app/features/scenes/scenes/index.tsx:37]: yea, this is not ok.  will likely leed to react errors

Comment (torkelo) [reply public/app/features/scenes/scenes/index.tsx:37]: I guess it's ok as this scene object has not activated (mounted yet)

File: public/app/features/scenes/components/Scene.tsx (hunk 2 above)
Comment (dprokop) [reply public/app/features/scenes/components/Scene.tsx]: yeah yeah, that's in no

File: public/app/features/scenes/scenes/index.tsx (hunk 1 above)
Comment (dprokop) [reply public/app/features/scenes/scenes/index.tsx]: Changed that - now there's a simple EmbeddedScene alternative

**Review by torkelo — APPROVED** [review]

**Pull request merged**